Automattic/wp-calypso
# Add inline comments to reader post cards.

Related to # pe7F0s-13p-p2

## Proposed Changes

* Shows the most recent comment by default below reader post cards, with the input to add comments. 
* A toggle to "view more comments" / "view less comments" to collapse/expand comments beyond the most recent.
* Another link to the full post page - "view more comments on full post" - if we have expanded and are showing less than all comments.
* Adds stats and tracks for new features (expanding to see more comments, and clicking to view all comments on the post). updates tracks for clicking reply, cancel reply, and submitting a comment to have an `is_inline_comment` property.

![inline-comments](https://github.com/Automattic/wp-calypso/assets/28742426/1742a60a-4a7b-4692-820a-a8b9741ba17e)
(note - we decided to get rid of the loading state in the above GIF)

## Testing Instructions



* Run this branch and test inline comments in various reader streams.
* Verify tracking events for reply/cancel buttons and posting a comment (and nested replies) have an `is_inline_comment` property added. Verify these dont happen in the comments section of the individual posts page.
* Verify tracking event exists for clicking to expand the comments section, and for clicking to "view all comments on post" (only happens on posts with many comments - eg Discover).

## Pre-merge Checklist
